tests/zfs-tests/tests/functional/cli_root/zpool_create/zpool_create_006_pos.ksh

```diff
@@ -49,6 +50,11 @@ function cleanup
 log_assert "Verify 'zpool create' succeed with keywords combination."
 log_onexit cleanup
 
+typeset disk="$($ECHO $DISKS | $NAWK '{print $1}')"
```

Comment:
Most of the setup scripts use this clever bit of shell `DISK=${DISKS%% *}` to get just the first DISK from the list of DISKS.  It would be good to use the same convention throughout.